src/main/kotlin/de/codecentric/hikaku/converter/openapi/OpenApiConverter.kt

```diff
@@ -96,6 +98,8 @@ class OpenApiConverter(private val openApiSpecification: String) : AbstractEndpo
             }
         }
 
+        @JvmStatic
+        @JvmName("usingFile")
```

Comment:
I like it, but could you add this to WadlConverter as well?

Reply:
done